src/main/webapp/support/group_chat.jsp

```diff
@@ -68,7 +68,7 @@
                     people that are paid to be readily available and answer your questions. It might take some time for
                     anyone to read your message.</p>
 
-                    <iframe src="candy.jsp" style="width:100%;height:400px;"></iframe>
+                    <iframe src="converse.jsp" style="width:100%;height:400px;"></iframe>
```

Comment:
The current state gives me a vertical scrollbar. It's gone by increasing the height to 420px:

`<iframe src="converse.jsp" style="width:100%;height: 420px;"></iframe>`